.config/forest.dic

```diff
@@ -33,3 +33,12 @@ tuple
 validator/S
 mempool
 sortition
+deserialize
+deserializing
+Deserializes
```

Comment:
We can use the /GS flags to add deserialize(ing/s) affixes to minimize words being duplicated with just endings changing

Reply:
Thanks!